### DISCUSSION TITLE
Empty mdtf.log

### DISCUSSION THREAD

**Discussion by drewmresnick** [NOAA-GFDL/MDTF-diagnostics#192] [Q&A]
I am working on a new POD, and am trying to debug. I see when I attempt to run the framework, in the output folder within /wkdir there is mdtf.log which I thought might give me some ideas as to why my POD fails. However it is blank. 

Might I have missed a step somewhere in setting up the POD which is preventing this log from working?

**Comment by wrongkindofdoctor** [comment]
Hi, @drewmresnick . If you can provide the commands you entered, terminal output, and the updated branch you are working on, we can help you solve the issue. I am moving this to the issue board since this a either a user error or framework problem. See #195.
